miden-lib/asm/kernels/transaction/lib/memory.masm

```diff
@@ -730,7 +730,7 @@ end
 #!
 #! Where:
 #! - num_procedures is the number of procedures contained in the account code.
-export.get_num_account_procedures
+export.get_num_procedures
     push.NUM_ACCT_PROCEDURES_PTR mem_load
 end
```

Comment:
Then we need to agree on naming because some procedures use `get_acct` and some procedures only use `get_` although both of them are accessing account data. 

#858

Reply:
Will tackle #858 next and refactor that at that point.

Reply:
Since we are going to refactor things in #858 - I don't see a reason to change the name here (especially since it makes things more ambiguous). I'll make a small commit to revert it.